packages/starter/create-react-app-starter/config-overrides.js

```diff
@@ -26,6 +26,7 @@ module.exports = function (config, env) {
                 assert: require.resolve('assert'),
                 buffer: require.resolve('buffer'),
                 stream: require.resolve('stream-browserify'),
+                crypto: require.resolve("crypto-browserify")
```

Comment:
I think if you merge the current master branch into the PR, this will go away because  it's already there: https://github.com/solana-labs/wallet-adapter/blob/e230958570f192786a27c57d5b39037b898e69dd/packages/starter/create-react-app-starter/config-overrides.js#L29

As above, it's not necessary to change the starter project at all.